packages/gestalt/src/Link.js

```diff
@@ -14,26 +20,83 @@ import useFocusVisible from './useFocusVisible.js';
+   */
   accessibilitySelected?: boolean,
+  /**
+   * Link is a wrapper around components (or children), most commonly [Text](https://gestalt.pinterest.systems/text), so that they become hiperlinks. See the [Link and Text](https://gestalt.pinterest.systems/link#Link-and-Text) variant for more information.
+   */
   children?: Node,
+  /**
+   * When `undeline` is supplied, Link's text is underlined on hover. See the [tapStyle and hoverStyle](https://gestalt.pinterest.systems/link#tapStyle-and-hoverStyle) variant for more information.
+   */
   hoverStyle?: 'none' | 'underline',
+  /**
+   * The URL that the hyperlink points to.
+   */
   href: string,
+  /**
+   * Id attribute of the anchor tag
+   */
   id?: string,
+  /**
+   * Properly positions Link relative to an inline element, such as [Text](https://gestalt.pinterest.systems/text), using the inline property.
+   */
   inline?: boolean,
+  /**
+   * Callback triggered when the user blurs Link.
+   */
   onBlur?: AbstractEventHandler<SyntheticFocusEvent<HTMLAnchorElement>>,
+  /**
+   * Callback fired when Link is clicked (pressed and released) with a mouse or keyboard. See [OnLinkNavigationProvider](https://gestalt.pinterest.systems/onlinknavigationprovider) to learn more about link navigation.
+   */
   onClick?: AbstractEventHandler<
     SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
     {| dangerouslyDisableOnNavigation: () => void |},
   >,
+  /**
+   * Callback triggered when the user focuses Link.
+   */
   onFocus?: AbstractEventHandler<SyntheticFocusEvent<HTMLAnchorElement>>,
+  /**
+   * Forward the ref to the underlying anchor element
+   */
+  ref?: Ref<'a'>,
+  /**
+   * Establishes the relationship of the linked URL. Use rel=”nofollow” for offsite links to inform search engines to ignore and not follow them
+   */
   rel?: 'none' | 'nofollow',
+  /**
+   * When supplied, Link acts a Tab. See the [Accessible Tab Link](https://gestalt.pinterest.systems/link#Accessible-Tab-Link) for more information.
+   */
   role?: 'tab',
+  /**
+   * Sets a border radius for Link. Select a rounding option that aligns with its children.
+
+Options are "circle" or "pill" for fully rounded corners or 0-8 representing the radius in boints.
+   */
   rounding?: Rounding,
+  /**
+   * When `compress` is supplied, Link is compressed on click. See the [tapStyle and hoverStyle](https://gestalt.pinterest.systems/link#tapStyle-and-hoverStyle) variant for more information.
+   */
   tapStyle?: 'none' | 'compress',
+  /**
+   * Define the frame or window to open the anchor defined on href:
+
+'null' opens the anchor in the same window.
+'blank' opens the anchor in a new window.
+'self' opens an anchor in the same frame.
+   */
   target?: null | 'self' | 'blank',
-  // private props to be internally used, therefore, not documented
-  disabled?: boolean,
+  /**
+   * Private prop
```

Comment:
the comment won't be displayed, but we can document for ourselves what this prop is doing

Reply:
Extended but disabled should be deprecated, it's an old prop from when InternalLink wasn't being used.

Reply:
Ah cool, hence not renaming to `_disabled`. I assume you'll take care of that deprecation in a future PR?